echo.js

```diff
@@ -1,7 +1,10 @@
 function echo(){
-  // ...
+  for (var i = 0; i < arguments.length; i++) {
+  	document.write(arguments[i] + "<br>");
+  };
 }
 
 echo();
 echo('bla');
 echo('foo', 'bar', 'baz');
+console.log("testing");
```

Comment:
This warning is a fluke – will try and get it sorted out for the next assignment. Sorry for any confusion!